drivers/lvutil.py

```diff
@@ -550,6 +578,7 @@ def setActiveVG(path, active):
     text = cmd_lvm([CMD_VGCHANGE, "-a" + val, path])
 
 
+@lvmretry
```

Comment:
Would it be better to decorate `cmd_lvm` instead of all its callers?

Reply:
> Would it be better to decorate `cmd_lvm` instead of all its callers?

If it was all the callers, yes, but it's not, so no.